egs/swbd/s5c/local/score_basic.sh

```diff
@@ -45,7 +46,7 @@ function filter_text {
   perl -e 'foreach $w (@ARGV) { $bad{$w} = 1; }
    while(<STDIN>) { @A  = split(" ", $_); $id = shift @A; print "$id ";
      foreach $a (@A) { if (!defined $bad{$a}) { print "$a "; }} print "\n"; }' \
-   '[NOISE]' '[LAUGHTER]' '[VOCALIZED-NOISE]' '<UNK>' '%HESITATION'
+   '[NOISE]' '[LAUGHTER]' '[VOCALIZED-NOISE]' '<UNK>' '%HESITATION' '[noise]' '[laughter]' '[vocalized-noise]' '<unk>' '%hesitation'
```

Comment:
OK.  just convert upper to lower case, no need for duplication.

On Thu, Mar 10, 2016 at 1:57 AM, Vimal Manohar <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/score_basic.sh
> <https://github.com/kaldi-asr/kaldi/pull/574#discussion_r55640367>:
>
> > @@ -45,7 +46,7 @@ function filter_text {
> >    perl -e 'foreach $w (@ARGV) { $bad{$w} = 1; }
> >     while(<STDIN>) { @A  = split(" ", $_); $id = shift @A; print "$id ";
> >       foreach $a (@A) { if (!defined $bad{$a}) { print "$a "; }} print "\n"; }' \
> > -   '[NOISE]' '[LAUGHTER]' '[VOCALIZED-NOISE]' '<UNK>' '%HESITATION'
> > +   '[NOISE]' '[LAUGHTER]' '[VOCALIZED-NOISE]' '<UNK>' '%HESITATION' '[noise]' '[laughter]' '[vocalized-noise]' '<unk>' '%hesitation'
>
> Will look into this. I'm not sure how much improvement this gave
> On Thu, Mar 10, 2016, 01:29 Daniel Povey <notifications@github.com>
> wrote: In egs/swbd/s5c/local/score_basic.sh <#574 (comment)
> <https://github.com/kaldi-asr/kaldi/pull/574#discussion_r55638802>>: > @@
> -45,7 +46,7 @@ function filter_text { > perl -e 'foreach $w (@ARGV
> <https://github.com/ARGV>) { $bad{$w} = 1; } > while(<STDIN>) { @A
> <https://github.com/A> = split(" ", $_); $id = shift @A
> <https://github.com/A>; print "$id "; > foreach $a (@A
> <https://github.com/A>) { if (!defined $bad{$a}) { print "$a "; }} print
> "\n"; }' \ > - '[NOISE]' '[LAUGHTER]' '[VOCALIZED-NOISE]' '<UNK>'
> '%HESITATION' > + '[NOISE]' '[LAUGHTER]' '[VOCALIZED-NOISE]' '<UNK>'
> '%HESITATION' '[noise]' '[laughter]' '[vocalized-noise]' '<unk>'
> '%hesitation' please act on my previous note on this. — Reply to this email
> directly or view it on GitHub <
> https://github.com/kaldi-asr/kaldi/pull/574/files#r55638802>.
> -- Vimal Manohar PhD Student Electrical & Computer Engineering Johns
> Hopkins University
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/574/files#r55640367>.
>